interop-testing/src/main/java/io/grpc/testing/integration/XdsTestServer.java

```diff
@@ -188,6 +188,7 @@ private void start() throws Exception {
               .addService(new XdsUpdateHealthServiceImpl(health))
               .addService(health.getHealthService())
               .addService(ProtoReflectionService.newInstance())
+              .addService(ChannelzService.newInstance(100))
```

Comment:
Nit: Not sure if 100 is actually important here or just an arbitrary value, but put into named constant since it's now used in two places?

Reply:
Makes sense